packages/connect/src/factory.ts

```diff
@@ -247,6 +247,8 @@ export const factory = ({
 
         firmwareUpdate: params => call({ ...params, method: 'firmwareUpdate' }),
 
+        firmwareUpdate_v2: params => call({ ...params, method: 'firmwareUpdate_v2' }),
```

Comment:
I will remove old firmwareUpdate in the last commit and replace it with the new one

Reply:
> or do we still need the old method? @komret

We don't, I think it can be deleted, I was afraid to do it.